tests/kafkatest/services/kafka.py

```diff
@@ -56,10 +58,12 @@ def start(self):
                 self.create_topic(topic_cfg)
 
     def start_node(self, node):
-        props_file = self.render('kafka.properties', node=node, broker_id=self.idx(node))
+        props_file = self.render('kafka.properties', node=node, broker_id=self.idx(node), interbroker_security_protocol=self.interbroker_security_protocol)
         self.logger.info("kafka.properties:")
         self.logger.info(props_file)
         node.account.create_file("/mnt/kafka.properties", props_file)
+        # Create keystore and truststore for SSL endpoint
+        SecurityConfig(node.account, SecurityConfig.SSL)
```

Comment:
Yes, Kafka has listeners with both PLAINTEXT and SSL in all tests. Clients choose the protocol with which to connect, and hence test one of the protocols. I will change Kafka to use only one listener to avoid confusion.